src/schema.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Joi = require('@hapi/joi');
+
+const defaultJoi = Joi.defaults(_schema =>
+  _schema.options({
+    stripUnknown: true
+  })
+);
+
+const _standupUpdateFile = defaultJoi.object().keys({
+  standupId: Joi.string().required(),
+
+  mimeType: Joi.string()
+    // Allow "audio/*" and "image/*"
+    .regex(/^(audio|image)\/[\w-]+/, 'mime-type')
+    .required(),
```

Comment:
We can limit the MIME-type to just be audio, since the schema is specific to a standup update.

Reply:
Alternatively, we can make the schema name more generic, and keep the image MIME type in (in case we'd want to upload an image for a standup).